### PR TITLE
Relerr

### PR DESCRIPTION
**Pull request by bubbleboy14**
use rel error event listeners.

replaced event (long-deprecated pyevent) references with rel.

### CODE REVIEW

File: dez/samples/http_client2.py [hunk 1]
```diff
@@ -1,23 +1,23 @@
 from dez.http.client import HTTPClientRequest, HTTPClient
-import event
+import rel
 import time
 
 def main(**kwargs):
-    event.initialize(["pyevent"])
+#    rel.initialize(["pyevent"])
```
Comment (DrMartinLutherXing) [inline dez/samples/http_client2.py]: just delete

**Review by DrMartinLutherXing — APPROVED** [review]

**Review by DrMartinLutherXing — APPROVED** [review]
reviewed

**Review by bubbleboy14 — COMMENTED** [review]
good call

**Comment by DrMartinLutherXing** [comment]
looks good

**Review by DrMartinLutherXing — APPROVED** [review]

**Review by DrMartinLutherXing — APPROVED** [review]

**Review by DrMartinLutherXing — APPROVED** [review]

File: dez/samples/http_client2.py (hunk 1 above)
Comment (bubbleboy14) [reply dez/samples/http_client2.py]: good call

**Pull request merged**